### ISSUE TITLE
script bekerja dengan sempurna

### ISSUE THREAD

**Issue by toastyy00** [ademulyanaDEV/t3rnv2#1]
![image](https://github.com/user-attachments/assets/fc7f0a06-9a83-471a-8629-58440f587991)
halo mas mulyana developer, ini bukannya kalo ga diganti malah assetnya kekirim ke address sampean?

**Comment by ademulyanaDEV** [comment]
Tidak, itu adalah metode yang digunakan untuk bridging (ABI). Tanpa metode tersebut, proses bridging mungkin akan salah arah dan menyebabkan error saat mengirim. Alamat tujuan yang sudah Anda tentukan di bawahnya adalah hasil akhir dari proses tersebut.

Untuk konsultasi lebih lanjut, Anda bisa menghubungi saya di Telegram melalui tautan ini: [@ademulyanaid](https://t.me/ademulyanaid).
